FAQ.md

```diff
@@ -81,3 +81,26 @@ If the client is trying to be defensive to nonconformant registries, and receive
 Mounting without having to specify `from`, also known as automatic mount origin discovery, requires the registry to determine whether or not a blob exists in any repository. 
 If the existence check for the blob is done first, an immediate failure will indicate the lack of presence of a blob. 
 On the other hand, if the registry needs to perform further work to determine if the blob can be accessed by the mounter, it could create an information disclosure risk, in leaking that presence of a blob with that digest in the registry.
+
+**Q: Why is support for new digest algorithms being added?**
+
+The default digest algorithm for OCI ecosystem has been _sha256_ and it has
+worked well. So why bother with additional digest algorithms?
+
+There are a couple of reasons (not in any particular order of importance):
```

Comment:
This is missing that blob uploads may be broken on some registries, particularly for post/patch/put requests. There is also no way to push a manifest with a tag by a client controlled algorithm, which means it's not possible to associate referrers with a tagged manifest unless the default algorithm is used.